src/MapExplorer.ls

```diff
@@ -23,12 +29,58 @@ package
     {
         private var _map:Map;
         private var _flyer:MapFlyer;
-        private var _list:List;
+        private var _listAttractions:List;
+        private var _listCategories:List;
+        private var _detailsView:Shape;
         private var _data:MapData;
         private var _timer:Timer;
 
+        private var _detailsTriggerTime:Number = NaN;
+
         public var onIdle:IdleDelegate;
 
+        public function gotoCategories():void
```

Comment:
I think if we have `goTo` below, this might look nicer as `goToCategories` or the other way around.